experimental/helm/charts/kserve-models-web-app/templates/deployment.yaml

```diff
@@ -29,22 +29,23 @@ spec:
         {{- include "kserve-models-web-app.kustomizeLabels" . | nindent 8 }}
         {{- if .Values.kubeflow.enabled }}
         {{- include "kserve-models-web-app.kserveLabels" . | nindent 8 }}
-        sidecar.istio.io/inject: "true"
         {{- end }}
         {{- with .Values.deployment.podLabels }}
         {{- toYaml . | nindent 8 }}
         {{- end }}
         {{- with .Values.app.labels }}
         {{- toYaml . | nindent 8 }}
         {{- end }}
-      {{- with .Values.deployment.podAnnotations }}
       annotations:
+        {{- if .Values.kubeflow.enabled }}
+        sidecar.istio.io/inject: "true"
+        {{- end }}
+        {{- with .Values.deployment.podAnnotations }}
         {{- toYaml . | nindent 8 }}
-      {{- end }}
-      {{- with .Values.app.annotations }}
-      annotations:
+        {{- end }}
+        {{- with .Values.app.annotations }}
```

Comment:
Why is it APP instead of the proper full word APPLICATION? Please fix that in all helm charts. Use proper long expressive and pronounceable words in the helm charts.